src/components/ExpenseForm.js

```diff
@@ -2,101 +2,158 @@ import { useState } from 'react';
+          required={false}
+          autoComplete="off"
+          maxLength="200"
+          placeholder="Add a description or comment"
+          value={inputValue.description}
+          onChange={event =>
+            setInputValue({ ...inputValue, description: event.target.value })
+          }
+        />
+        <label htmlFor="amount">Amount (€):</label>
+        <input
+          type="text"
+          name="amount"
+          inputMode="numeric"
+          id="amount"
+          required={true}
+          autoComplete="off"
+          pattern="^\d*(\.\d{0,2})?$"
+          maxLength="9"
+          value={inputValue.amount}
+          placeholder="Use '.' as a decimal separator, e.g. 12.45"
+          onChange={event =>
+            setInputValue({
+              ...inputValue,
+              amount: event.target.value,
+            })
+          }
+        />
+        <label htmlFor="date">Date:</label>
+        <input
+          type="date"
+          name="date"
+          id="date"
+          required={true}
+          value={inputValue.date}
+          onChange={event =>
+            setInputValue({
+              ...inputValue,
+              date: event.target.value,
+            })
+          }
+        />
+        <p>
+          Who was the expense paid <span style={{ color: 'red' }}>by?</span>
+        </p>
+
+        {members.map((member, index) => (
+          <div key={index}>
+            <label htmlFor={index}>{member}:</label>
+            <input
+              type="radio"
+              name="paidby"
+              id={index}
+              required={true}
+              value={inputValue.paidby}
+              onChange={() =>
+                setInputValue({
+                  ...inputValue,
+                  paidby: member,
+                })
+              }
+            />
+          </div>
+        ))}
+        <p>
+          Who was the expense paid <span style={{ color: 'red' }}>for?</span>
+        </p>
+        {members.map((member, index) => (
+          <div key={index}>
+            <label htmlFor={index}>{member}:</label>
+            <input
+              type="checkbox"
+              name="paidfor"
+              id={index}
+              onChange={event => {
+                if (event.target.checked) {
+                  if (!inputValue.paidfor.includes(member)) {
+                    setInputValue(() => ({
+                      ...inputValue,
+                      paidfor: [...inputValue.paidfor, member],
+                    }));
+                  }
+                } else {
+                  setInputValue(() => ({
+                    ...inputValue,
+                    paidfor: inputValue.paidfor.filter(name => name !== member),
+                  }));
+                }
+              }}
+            />
+          </div>
+        ))}
+        <button>Add expense</button>
+      </Form>
+    );
+  }
+  return <MemberError>Please add members on the main page first</MemberError>;
```

Comment:
Is this component specific for member errors? If not you could think about renaming it so you can display other error messages with it as well!